docs/pages/components/panel.md

```diff
@@ -0,0 +1,161 @@
+---
+title: Panel
+id: expand-panel
+keywords: panel
+sidebar: left-navigation-sidebar
+toc: false
+permalink: components/panel.html
+folder: components
+---
+The panel is a container for grouping and displaying information. It can be collapsed to save space on the screen.
+{: .docs-intro}
+
+
+Use the panel if:
+<ul> 
+    <li>You need to group or display information.</li>
+    <li>You want to give users the option of hiding this information.</li>
+    <li>You want to show additional information on demand (for example, a panel could show optional input fields for an advanced search).</li>
+</ul>
+Do not use the panel if:
+<ul>
+    <li>You are designing an object page. Never use panels in the object page content area.</li>
+</ul>
+
+
+## Expandable Panel
+Expandable panels are much like fixed panels, except their content can be expanded and collapsed (including the info toolbar, if available).
+{% capture expand-panel %}
+<div class="fd-expand-panel">
+    <div class="fd-expand-panel__header">
+        <div class="fd-expand-panel__expand">
+            <button class="fd-button fd-button--transparent fd-expand-panel__button" aria-expanded="false" aria-haspopup="true" aria-controls="jhqDKYdf"></button>
+        </div>
+        <h5 class="fd-expand-panel__title">Panel Header</h5>
+        <div class="fd-toolbar fd-toolbar--clear fd-toolbar--transparent">
+            <span class="fd-toolbar__spacer fd-toolbar__spacer--auto"> </span>
+            <div class="fd-segmented-button" role="group" aria-label="Group label">
+              <button class="fd-button is-selected" aria-pressed="true">Left</button>
+              <button class="fd-button">Middle</button>
+              <button class="fd-button">Right</button>
+            </div>
+        </div>
+    </div>
+    <div role="region" aria-labelledby="jhqDKYdf" class="fd-expand-panel__content" aria-hidden="true" id="jhqDKYdf">
```

Comment:
The `aria-labelledby` should reference an element that labels the element, not itself. In this case, I'm assuming it's one of the Left / Middle / Right buttons, and this switches depending which one is active (hard to tell from the example).

So an `id` should be added to the segmented buttons, and the `aria-labelledby` on the region should be whichever button is active.